test_scripts/RC/OnRCStatus/commonOnRCStatus.lua

```diff
@@ -140,30 +140,58 @@ function m.postconditions()
   restorePreloadedPT()
 end
 
-function m.setModuleStatus(pFreeMod, pAllocatedMod, pModule)
+local function setAllocationState(pAllocArrays, pAllocApp, pModule)
+    for i=1,#pAllocArrays do
+      for key, value in pairs(pAllocArrays[i]) do
+        if pModule == value then
+          table.remove(pAllocArrays[i], key)
+        end
+      end
+    end
+    table.insert(pAllocArrays[pAllocApp], pModule)
+  return pAllocArrays
+end
+
+function m.setModuleStatus(pFreeMod, pAllocatedMod, pModule, pAllocApp)
+  if not pAllocApp then pAllocApp = 1 end
+  local ModulesStatusAllocated = { }
   local ModulesStatus = { }
-  table.insert(pAllocatedMod, pModule)
+  AllocatedMod = setAllocationState(pAllocatedMod, pAllocApp, pModule)
```

Comment:
@GetmanetsIrina Seems I was wrong: ```AllocatedMod``` and ```pAllocatedMod``` are different variables
In this way instead of ```AllocatedMod``` ```pAllocatedMod``` can be used